site/en/blog/chips-origin-trial-extended/index.md

```diff
@@ -2,13 +2,15 @@
 layout: 'layouts/blog-post.njk'
 title: Cookies Having Independent State (CHIPS) origin trial extended
 description: >
-  CHIPS is a Privacy Sandbox proposal that introduces a mechanism to opt-in
-  to having third-party cookies partitioned by top-level site. The origin trial
-  that started in Chrome 100, will now be available until the release of Chrome 105,
-  scheduled for August 30, 2022.
+  CHIPS is a Privacy Sandbox proposal that introduces a mechanism to opt-in to
+  having third-party cookies partitioned by top-level site. The origin trial
+  that started in Chrome 100, will now be available until the release of Chrome
```

Comment:
Due to the recent outage, we can only enable the OT in M106 up until Oct. 25th. So we should probably update the lower bound milestone?

Reply:
To clarify, the OT ends on Oct. 25th regardless of the 107 release?

Reply:
Wording clarified.